### ISSUE TITLE
Catalina 10.15.7 Update

### ISSUE THREAD

**Issue by lindam33** [ExistentialAudio/BlackHole#225]
I recently downloaded Blackhole and used it with success in Quicktime. I updated my OS to Catalina 10.15.7 this morning, and now when I try to record internal video with Quicktime I don't have any sound. Are there issues with the latest OS update, or is it me :) ?

**Comment by devinroth** [comment]
Let me check.

**Comment by devinroth** [comment]
Shouldn't be any issues. Is QuickTime still listed in microphone permissions? Are you using 2ch version? If not I recommend using it with Quicktime? You can alway try reinstalling. I've heard that that fixes stuff sometimes.

**Comment by lindam33** [comment]
Thanks Devin.  The issue arose when I discovered I could not control the volume on my laptop when Blackhole was enabled.  Once I disabled it, I had volume control on my laptop, but then then Blackhole didn’t record internally regardless of the settings.  This happened at the same time my OS was upgraded, so I wasn’t sure of the issue origins.

I reinstalled Blackhole and it works again (yay!) but once again I cannot adjust volume controls on my laptop. Just thought I’d send you this update in case you find it happens with others.

Have a great day,
Linda
